### PR TITLE
luci-mod-status: Adding scroll buttons on syslog and kernellog status pages

### PR DESCRIPTION
**Pull request by Ramon00**
My log files can grow rather big, typically if i look at the log files then i want to look at the bottom, not the top, so I was missing a scroll down button. 

So I added these:
image

image

Please let me know if you have any comments or suggestions.

### CODE REVIEW

**Comment by systemcrash** [comment]
Might I suggest Tail and Head, especially for those who might already be
familiar with those utilities that do exactly these things.

**Comment by Ramon00** [comment]
@systemcrash yes i use that, but sometimes I want to just check the log from my phone, then i have to scroll all the way down... Anyway if you think this is not a good addition then fine, just reject this PR, i will just use it on my own openwrt installs, just thought it might be useful for somebody else as well. Not sure though why we would not want to make the LuCi logfile page more useful... Or do you mean I can use tail from LuCi? Please tell me how.

**Comment by hnyman** [comment]
> do you mean I can use tail from LuCi? Please tell me how.

Example of reversing log.
https://forum.openwrt.org/t/status-system-log-new-text-should-be-at-the-top-not-bottom/151740/4

**Comment by Ramon00** [comment]
@systemcrash Which package is that in?

**Comment by jow-** [comment]
The button style looks wrong

**Comment by Ramon00** [comment]
@jow- if you can show me an example what button style you would like, then I can adjust it, np. I just put in the most simple button i could find

**Comment by Ramon00** [comment]
@jow- Like this perhaps: 
image

File: modules/luci-mod-status/htdocs/luci-static/resources/view/status/syslog.js [hunk 1]
```diff
@@ -21,16 +21,37 @@ return view.extend({
...
+			}, _('Scroll down')
+		);
+		scrollDownButton.addEventListener('click', function() {
+			window.scrollTo(0, document.body.scrollHeight);
+		});
+
+		var scrollUpButton = E('button', { 
+				'id' : 'scrollUpButton',
+				'class': 'cbi-button cbi-button-neutral'
+			}, _('Scroll up')
```
Comment (systemcrash) [inline modules/luci-mod-status/htdocs/luci-static/resources/view/status/syslog.js:37]: You could change instances to just `Tail` - a simpler way of saying more concretely `Scroll to bottom` or `Scroll to tail`, which is more definitive than just `Scroll down`.

Likewise for the other button names. You decide, but put something more concrete.

**Review by systemcrash — CHANGES_REQUESTED** [review]

File: modules/luci-mod-status/htdocs/luci-static/resources/view/status/syslog.js (hunk 1 above)
Comment (Ramon00) [reply modules/luci-mod-status/htdocs/luci-static/resources/view/status/syslog.js:37]: To understand correctly, you just want to name the button "tail" and "head"? If you think that is more direct then im fine with that.

Comment (systemcrash) [reply modules/luci-mod-status/htdocs/luci-static/resources/view/status/syslog.js:37]: It's short and easier to translate than three words. :)

Maybe `Scroll to tail` is best - more context for the i18n people.

Comment (Ramon00) [reply modules/luci-mod-status/htdocs/luci-static/resources/view/status/syslog.js:37]: yeah i can do that, i can also add a translation hint separately.

**Review by systemcrash — APPROVED** [review]

**Comment by systemcrash** [comment]
I just tested this, and it seems to work fine. Can you put a bit more padding between scroll to head button and the log field?

**Comment by Ramon00** [comment]
@systemcrash I encapsulated the bottom button now in a div and added 20px padding below the button.  The padding between the button and the logfile itself is in the 'textarea' already, so i cannot really remove it. I can also add paddint to the top button either at above and/or below the button.  Just a matter of what the preference is i suppose

**Comment by systemcrash** [comment]
> @systemcrash I encapsulated the bottom button now in a div and added 20px padding below the button. The padding between the button and the logfile itself is in the 'textarea' already, so i cannot really remove it. I can also add paddint to the top button either at above and/or below the button. Just a matter of what the preference is i suppose

Below the botton at the top, so it's not adjoining the log field

**Comment by Ramon00** [comment]
Ah you didnt mean the scroll to head button, but the scroll to tail button. Something like this? 
image
and 
image

**Comment by systemcrash** [comment]
Exactly so

**Comment by Ramon00** [comment]
ok cool.

**Comment by systemcrash** [comment]
Please rebase onto current master.

**Comment by Ramon00** [comment]
> Please rebase onto current master.

done

**Comment by systemcrash** [comment]
Thank you, nicely done :)

**Comment by Ramon00** [comment]
And thank you for the review and the suggestions :)

**Pull request merged**